consent/self-service-portal/main.go

```diff
@@ -41,6 +41,18 @@ type Config struct {
 	IntrospectClientID          string        `env:"INTROSPECT_CLIENT_ID" envDefault:"bv2dkff8mll9cf6pvd6g"`
 	IntrospectClientSecret      string        `env:"INTROSPECT_CLIENT_SECRET" envDefault:"KThGH68f-gMC4cscGLFeOpIU4EYriYhKspOV9IwHbnw"`
 	IntrospectIssuerURL         *url.URL      `env:"INTROSPECT_ISSUER_URL,required"`
+	BankClientConfig            BankClientConfig
+}
+
+type BankClientConfig struct {
+	URL          *url.URL `env:"BANK_URL,required"`      // v
+	AccountsURL  *url.URL `env:"BANK_ACCOUNTS_ENDPOINT"` // v
+	TokenURL     string   `env:"BANK_CLIENT_TOKEN_URL"`  // v
+	ClientID     string   `env:"BANK_CLIENT_ID"`         // v
+	ClientSecret string   `env:"BANK_CLIENT_SECRET"`     // v
+	Scopes       []string `env:"BANK_CLIENT_SCOPES"`     // v
+	CertFile     string   `env:"BANK_CLIENT_CERT_FILE"`  // x
+	KeyFile      string   `env:"BANK_CLIENT_KEY_FILE"`   // x
```

Comment:
I don't understand `// v, x` comments, please clarify

Reply:
accidentally left that. Its cleaned up now